docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+            <td><p>List&lt;Enum&gt;</p></td>
+            <td>A semicolon-separated list of Compression Type. Different levels can have different compression policies. In many cases, lower levels use fast compression algorithms, while higher levels with more data use slower but more effective compression algorithms. The N th element in the List corresponds to the compression type of the level N-1When <code class="highlighter-rouge">state.backend.forst.compaction.level.use-dynamic-size</code> is true, compression_per_level[0] still determines L0, but other elements are based on the base level and may not match the level seen in the info log<br />Note: If the List size is smaller than the level number, the undefined lower level uses the last Compression Type in the List<br />Some commonly used compression algorithms for candidates include <code class="highlighter-rouge">NO_COMPRESSION</code> ,<code class="highlighter-rouge">SNAPPY_COMPRESSION</code> and <code class="highlighter-rouge">LZ4_COMPRESSION</code><br />The default value is <code class="highlighter-rouge">SNAPPY_COMPRESSION</code>, which means that all data uses the Snappy compression algorithm.Likewise, if set to <code class="highlighter-rouge">NO_COMPRESSION</code> , means that all data is not compressed, which will achieve faster speed but will bring some space amplification.In addition, if we need to consider both spatial amplification and performance, we can also set it to '<code class="highlighter-rouge">NO_COMPRESSION</code>;<code class="highlighter-rouge">NO_COMPRESSION</code>;<code class="highlighter-rouge">LZ4_COMPRESSION</code>', which means that L0 and L1 data will not be compressed, and other data will be compressed using LZ4.<br /><br />Possible values:<ul><li>"NO_COMPRESSION"</li><li>"SNAPPY_COMPRESSION"</li><li>"ZLIB_COMPRESSION"</li><li>"BZLIB2_COMPRESSION"</li><li>"LZ4_COMPRESSION"</li><li>"LZ4HC_COMPRESSION"</li><li>"XPRESS_COMPRESSION"</li><li>"ZSTD_COMPRESSION"</li><li>"DISABLE_COMPRESSION_OPTION"</li></ul></td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.files.open</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Integer</td>
+            <td>The maximum number of open files (per stateful operator) that can be used by the DB, '-1' means no limit. The default value is '-1'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.log.dir</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>The directory for ForSt's information logging files. If empty (Flink default setting), log files will be in the same directory as the Flink log. If non-empty, this directory will be used and the data directory's absolute path will be used as the prefix of the log file name. If setting this option as a non-existing location, e.g '/dev/null', ForSt will then create the log under its own database folder as before.</td>
```

Comment:
how would this work in Kubenetes when log forwarding could be in place.

Reply:
> What does as before mean here?
It is misleading, removed.

> how would this work in Kubenetes when log forwarding could be in place.
I'm not sure what you mean here....